### PR TITLE
AuxTel dome Emergency Shutdown Procedure

### PR DESCRIPTION
**Pull request by kpenaramirez**
Can you please review the draft version of the Emergency Shutdown Procedure for AuxTel. For reference it is based on the [related confluence page](https://confluence.lsstcorp.org/display/OOD/ATDome+shutter+fails+to+close) and it would look like [this](https://obs-ops.lsst.io/v/SITCOM-1361/AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.html).

### CODE REVIEW

File: AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+.. warning::
+    If you are dealing with a dome shutter emergency shutdown. Proceed calmly  to keep the equipment safe.
+
+.. _Title-of-Procedure-Critical-Step-1:
+
+Shutter Closure
+---------------
+
+1. Go up to the AuxTel dome (second floor).  Locate the control box at the top of the stairs.  The top black button should close the shutter.
```
Comment (craiglagegit) [inline AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst:74]: Add this to this line:  Note that the safety gate at the bottom of the stairs needs to be closed and locked in order for this button to operate,

**Review by craiglagegit — APPROVED** [review]

File: AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  :name: Switches inside AuxTel cRIO
+
+With **EXTREME CAUTION** slide the switch labeled "CLOSE" (yellow arrow) progresivily to the left. This is a low-level control switch and there are no limit switches or fail-safes. It is best to stop the shutter movement about 1 cm before it is fully closed to prevent you over drive the shutter until it bangs into the lower shutter or cause damage on the drive.
+
+.. _Title-of-Procedure-Critical-Step-2:
+
+Shutter Closure without Power
+-----------------------------
+
+1. In the absence of electrical power, the shutter must be closed manually. On AuxTel first floor there is a crank that fits into a ring at the top of the dome. Rotating the ring manually with the crank assistance close the shutter. Note this is a long and physically demaning procedure and the hand crank just does not have enough torque to lift the weight of the shutter if it is all the way open.
```
Comment (christensen-e) [inline AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst:94]: spelling - "demanding"

File: AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+
```
Comment (christensen-e) [inline AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst:95]: Add this line: If you cannot safely insert the crank into the drive motor and operate the crank, then remove the crank and do not proceed.

File: AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+- The emergency situation has been clearly notified to the relevant personnel and management.
+- Steps to eliminate or minimize the risk to the instruments and hardware are being implemented.
+- Eventually, the dome shutter is closed and all equipment is safe.
+
+.. _Title-of-Procedure-Procedure-Steps:
+Procedure Steps
+===============
+
+.. todo::
+   Please use the standard safety measures when visiting the dome enclosure. **Safety of personnel always goes first.**
```
Comment (christensen-e) [inline AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst:58]: This comment is not visible on the page https://obs-ops.lsst.io/v/SITCOM-1361/AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.html - am I looking at the right branch?

File: AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+In the event of an emergency dome closure, the dome shutter won't be able to be operated from the standard CSC controls. The AuxTel dome would need to be closed manually. In case of sudden inclement weather, be very cautious and start closing early.
+
+.. warning::
+    If you are dealing with a dome shutter emergency shutdown. Proceed calmly  to keep the equipment safe.
+
+.. _Title-of-Procedure-Critical-Step-1:
+
+Shutter Closure
+---------------
+
```
Comment (christensen-e) [inline AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst:73]: I think there needs to be a first step, which is to communicate on #summit-announce that people are going to attempt to close the AuxTel dome via the emergency procedure.

**Review by christensen-e — CHANGES_REQUESTED** [review]

File: AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst (hunk 4 above)
Comment (kpenaramirez) [reply AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst:58]: Added to the existing warning.

File: AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst (hunk 5 above)
Comment (kpenaramirez) [reply AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst:73]: Added.

File: AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst (hunk 1 above)
Comment (kpenaramirez) [reply AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst:74]: Added.

File: AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst (hunk 2 above)
Comment (kpenaramirez) [reply AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst:94]: Thanks!

File: AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst (hunk 3 above)
Comment (kpenaramirez) [reply AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst:95]: Added.

**Comment by kpenaramirez** [comment]
Thank for your comments. The procedure has been edited as requested. A final note regarding contingency was also added, it reads:
_If the procedure was not successful, report the issue on the *#summit_announce* channel and/or activate the Out of hours support._

**Pull request merged**